views/signup.ejs

```diff
@@ -27,6 +27,10 @@
   
 </form>
 
+<form method = 'GET' action = 'login'>
```

Comment:
Is this just to catch to people who've navigated to sign-up by mistake but already have an account?

Reply:
It is indeed.